### PR TITLE
runtime docs

### PR DESCRIPTION
**Pull request by hzoo**
For https://github.com/babel/babel/pull/8266

Preview:
- https://deploy-preview-1714--babel.netlify.com/docs/en/next/babel-runtime
- https://deploy-preview-1714--babel.netlify.com/docs/en/next/babel-plugin-transform-runtime

### CODE REVIEW

**Comment by hzoo** [comment]
Deploy preview for *babel-new* ready!

Built with commit 6952c372c54556c4eb4e70f9ac3eb50efecc7ad5

https://deploy-preview-1714--babel-new.netlify.com

**Comment by hzoo** [comment]
Deploy preview for *babel-new* ready!

Built with commit 7c8598762a13e7ad88d893f0ef154e3e942942f4

https://deploy-preview-1714--babel-new.netlify.com

File: docs/plugin-transform-runtime.md [hunk 1]
```diff
@@ -4,37 +4,37 @@ title: babel-plugin-transform-runtime
...
-
-In most cases, you should install `@babel/plugin-transform-runtime` as a development dependency (with `--save-dev`).
+Install it as development dependency.
 
 ```sh
 npm install --save-dev @babel/plugin-transform-runtime
 ```
 
-and `@babel/runtime` as a production dependency (with `--save`).
+and [`@babel/runtime`](runtime.md) as a production dependency (since it's for the "runtime").
```
Comment (Andarist) [inline docs/plugin-transform-runtime.md]: I'm wondering - can this somehow be made obvious? Like maybe we should validate it? This thing is really confusing for newcomers and it's easy to miss installing this

Comment (hzoo) [reply docs/plugin-transform-runtime.md]: Logan was thinking about what we can do here but not sure what we can do for v7.

**Pull request merged**